delirium/dns/models/database.py

```diff
@@ -31,15 +32,11 @@ def init_db(path):
     c.execute(QUERY_CREATE_TABLE)
     return c
 
-def create_cache(addr_range, duration, cache_type, path=DEFAULT_DB_PATH):
+def create_cache(addr_range, duration, path=DEFAULT_DB_PATH):
```

Comment:
This factory method is no longer required since there's only one type of cache now

Reply:
Recommend fixing as part of #24